projects/packages/sync/changelog/add-phan-wordpress-globals

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: fixed
+Comment: Add int casts on various `$wpdb->get_var( "SELECT COUNT(*) ..." )` to make Phan happy
```

Comment:
One side effect of this is that an error would return `null`. Casting to `int` would result in 0 (effectively suppressing the error), which may have undesired results if there's error-catching downstream.

Reply:
In several of these cases it's doing that implicitly anyway. And in the rest the phpdoc was giving no indication of null being a possibility.

WordPress in general doesn't seem to consider DB errors very often, even in core I've seen places where they don't bother to differentiate "no results" from "error happened". It'd probably be better if they were to start throwing exceptions for errors instead of returning null/false. I'd not be surprised if there's a 15-year-old trac issue about that that no one will ever do.